src/Components/Contact.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import "../stylesheets/Contact.scss";
+
+const Contact = () => {
+  return (
+    <div className="contact-page__container">
+      <h1 className="contact__heading--primary">Contact Us</h1>
+      {/* Left side */}
+      <div className="contact-page__body">
+        <div className="contact-form__container">
+          <form>
+            <p>
+              <span className="contact-form__mandatory-field" />
+              Indicates required field
+            </p>
+            <div className="contact-form__input-box--text-short">
+              <label for="firstName" className="contact-form__label">
```

Comment:
```suggestion
              <label htmlFor="firstName" className="contact-form__label">
```
There are a few different places that this needs to be changed so that it isn't throwing an error in the console.